src/Hooks/CommandStartingListener.php

```diff
@@ -62,26 +65,26 @@ private function registerJobHooks(): void
         $this->nightwatch->configureForJobs();
 
         /**
+         * @see \Laravel\Nightwatch\Core::ingest()
          * @see \Laravel\Nightwatch\State\CommandState::reset()
-         */
-        $this->events->listen(JobPopping::class, (new JobPoppingListener($this->nightwatch))(...));
-
-        /**
          * @see \Laravel\Nightwatch\State\CommandState::$timestamp
          * @see \Laravel\Nightwatch\State\CommandState::$id
          */
-        $this->events->listen(JobProcessing::class, (new JobProcessingListener($this->nightwatch))(...));
-
-        /**
-         * @see \Laravel\Nightwatch\Records\Exception
-         */
-        $this->events->listen(JobExceptionOccurred::class, (new JobExceptionOccurredListener($this->nightwatch))(...));
+        $this->events->listen([
+            Looping::class,
+            JobPopping::class,
+            JobProcessing::class,
+            WorkerStopping::class,
+        ], (new WorkerEventListener($this->nightwatch))(...));
 
         /**
          * @see \Laravel\Nightwatch\Records\JobAttempt
-         * @see \Laravel\Nightwatch\Core::ingest()
          */
-        $this->events->listen(JobAttempted::class, (new JobAttemptedListener($this->nightwatch))(...));
+        $this->events->listen([
+            JobProcessed::class,
+            JobReleasedAfterException::class,
+            JobFailed::class,
+        ], (new JobAttemptListener($this->nightwatch))(...));
```

Comment:
We listen for the `JobProcessed`, `JobReleasedAfterException`, and `JobFailed` events as the `JobAttempted` event is not available in older versions of the framework.